composites/Plugin/Shared/components/YoastButton.js

```diff
@@ -86,7 +86,12 @@ YoastButtonBase.propTypes = {
 	className: PropTypes.string,
 	onClick: PropTypes.func,
 	type: PropTypes.string,
-	children: PropTypes.node,
+	isExpanded: PropTypes.bool,
+	children: PropTypes.oneOfType( [
+		PropTypes.arrayOf( PropTypes.node ),
+		PropTypes.node,
+		PropTypes.string,
```

Comment:
Aren't strings nodes?

Reply:
Not sure in React terms. In JavaScript terms, no they're not the same thing. In DOM terms, `text` is a node. See also `makeOutboundLink`:
```
		children: PropTypes.oneOfType( [
			PropTypes.string,
			PropTypes.node,
		] ),
```

Reply:
According to Facebook's React docs:
```
// Anything that can be rendered: numbers, strings, elements or an array
// (or fragment) containing these types.
  optionalNode: PropTypes.node,
```

Reply:
Interesting, although sounds a bit weird to me. So according to the React docs we could  instead of:
`width: PropTypes.number,` also `width: PropTypes.node,` ?

Anyways, I've updated the prop types also for `makeOutboundLink` for consistency, please test with the updated test code sample.